docs/overview/office-add-in-availability.md

```diff
@@ -51,6 +51,34 @@ To work as expected, your Office Add-in might depend on a specific Office host,
         - TextBindings<br>
         - TextCoercion</td>
   </tr>
+  <tr>
+    <td>Office 365</td>
```

Comment:
Actually, I think it may make sense that we don't have sections for Office 365: Everything is eventually supported in it, but what is supported depends on the build you've got, which depends on your channel. So its not very useful to just have a listing for "Office 365" in general. Make sense?

Reply:
Isn't the same true for Office Online?